client/src/util/styles.ts

```diff
@@ -0,0 +1,51 @@
+import { css, DefaultTheme } from "styled-components";
+
+const purple = ({ opacity = 1 } = {}) => `rgba(39, 0, 84, ${opacity})`;
```

Comment:
Can you annotated these functions with their return types?

like:

```ts
const purple = ({ opacity = 1 } = {}): string => `rgba(39, 0, 84, ${opacity})`;
```

Reply:
Resolved